native/packages/react-native-bpk-component-button/src/layout/BpkButtonContainer.android.js

```diff
@@ -0,0 +1,101 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2017 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  View,
+  TouchableNativeFeedback,
+} from 'react-native';
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import {
+  getStyleForElement,
+  getAndroidBackgroundColour,
+  iconPropType,
+  themePropType,
+} from './../BpkButton-utils';
+
+const BpkButtonContainer = (props) => {
+  const {
+     type,
+    title,
+    icon,
+    iconOnly,
+    onPress,
+    large,
+    disabled,
+    accessibilityLabel,
+    style,
+    children,
+    ...rest
+   } = props;
+
+  const theme = props.theme;
+  const accessibilityTraits = ['button'];
+  if (disabled) {
+    accessibilityTraits.push('disabled');
+  }
+
+  return (
+    <View
+      style={[getStyleForElement('container', props), style]}
+    >
+      <TouchableNativeFeedback
+        disabled={disabled}
+        onPress={onPress}
+        background={TouchableNativeFeedback.SelectableBackgroundBorderless()}
+        {...rest}
+      >
+        <View
+          style={[getStyleForElement('button', props), getAndroidBackgroundColour(theme, props)]}
+        >
+          {children}
+        </View>
+      </TouchableNativeFeedback>
+    </View>
+  );
+};
+
+const propTypes = {
```

Comment:
these prop types can be assigned directly to `BpkButtonContainer.propTypes`

Reply:
Agreed - I think this is just an oversight as in the original `BpkButton.js` file we had to do this so it could be exported and accessed from a test.

Reply:
In `BpkButton.js` we're exporting those propTypes, in the container no, so it was my oversight 😄